File: easycloud-cluster/src/main/java/dev/easycloud/service/configuration/LocalConfiguration.java
package dev.easycloud.service.configuration;

import dev.easycloud.service.file.resources.FileEntity;
import lombok.Getter;
import lombok.Setter;

import java.util.Locale;

@Getter
@Setter
@FileEntity(name = "local")
public final class LocalConfiguration {
    private Locale language = Locale.ENGLISH;
    private boolean announceUpdates = true;
    private int clusterPort = 5200;
    private int proxyPort = 25565;
    private int startingSameTime = 3;
    private int dynamicPercentage = 80;
}

File: easycloud-cluster/src/main/java/dev/easycloud/service/configuration/LocalConfiguration.kt
package dev.easycloud.service.configuration

import dev.easycloud.service.configuration.resources.ConfigurationEntity
import java.util.Locale

@ConfigurationEntity(name = "local")
class LocalConfiguration(
    var language: Locale = Locale.ENGLISH,
    var announceUpdates: Boolean = true,
    var clusterPort: Int = 8080,
    var proxyPort: Int = 25565,
    var startingSameTime: Int = 3,
    var dynamicPercentage: Int = 80
)